src/config.js

```diff
@@ -1,4 +1,4 @@
-let DEBUG = false;
+let DEBUG = true;
```

Comment:
Should this be `false`?
```suggestion
let DEBUG = false;
```

Reply:
👍🏻